test/printf-test.cc

```diff
@@ -589,7 +590,8 @@ class custom_printf_arg_formatter : public formatter_t {
       if (round(value * pow(10, specs()->precision)) == 0.0) value = 0;
   return formatter_t::operator()(value);
 }
-};
+}
+;
```

Comment:
Could you remove ; while at it? =)

Reply:
Can't, since it follows a class definition :)  Not sure why clang-format wants to put it on a separate line.